django_project/certification/migrations/0013_auto_20171014_1045.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Your code should not be touching this file right?

Reply:
no @timlinux

Reply:
I had a few migration breaks but its okay now.

Reply:
So can you remove this from your changes then?